packages/pcd/ethereum-group-pcd/test/EthereumGroupPCD.spec.ts

```diff
@@ -131,7 +131,6 @@ async function happyPathEthGroupPCD(
 }
 
 describe("Ethereum Group PCD", function () {
-  this.timeout(60 * 1000 * 10);
```

Comment:
i think this test actually does need this really long timeout, and it's perhaps the only one. i'd be fine with either ignoring the lint here or increasing the default timeout from 1 min to even longer in the root `.mocharc.js`